goldstone/core/models.py

```diff
@@ -2696,6 +2701,69 @@ def build_alert_template(self, hits):
         return msg_params_dict
 
 
+class AlertSearchSQLQuery(AlertSearch):
+
+    db_table = models.CharField(max_length=64,
+                                default='compliance_vulnerability',
+                                help_text='database to be searched')
+
+    db_col_filter = models.CharField(max_length=64, default='created',
```

Comment:
I don't understand what this column is for. The default leads me to believe that it is the key that would be used for `search_recent`.  If so, we already have such a field from the SavedSearch superclass.